src/chase.rs

```diff
@@ -49,7 +51,7 @@ pub struct Rel(pub String);
 
 impl Rel {
     pub fn new(name: &str) -> Rel {
-        Rel(name.to_string())
+        Rel(name.to_owned())
```

Comment:
There used to be a performance difference between `to_owned` and `to_string`, but I've been told that there is no difference between the two nowadays.

Reply:
Good to know. I made the change based on what I read online but probably older material.